src/sentry/api/bases/organization_events.py

```diff
@@ -387,10 +413,12 @@ def serialize_multiple_axis(
 
 
 class KeyTransactionBase(OrganizationEventsV2EndpointBase):
-    def has_feature(self, request, organization):
-        return features.has("organizations:performance-view", organization, actor=request.user)
+    def has_feature(self, organization: Organization, request: Request) -> bool:
```

Comment:
This signature was backwards compared to the parent, had to change this plus calls to it in discover_key_transactions.py